kratos/utilities/normal_calculation_utils.cpp

```diff
@@ -591,39 +591,39 @@ void NormalCalculationUtils::CalculateNormalsUsingGenericAlgorithm(
     // Calculate normals in entities
     CalculateNormalsInContainer<TContainerType>(rModelPart, rNormalVariable);
 
-    // Declare auxiliar coordinates
-    Point::CoordinatesArrayType aux_coords;
-
-    auto& r_entities_array = GetContainer<TContainerType>(rModelPart);
-    const auto it_entity_begin = r_entities_array.begin();
-
     const std::function<array_1d<double, 3>(const GeometryType&, const GeometryType::CoordinatesArrayType&, const double)> retrieve_normal_unit_normal =
     [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) -> array_1d<double, 3> {return rGeometry.UnitNormal(rLocalCoordinates);};
     const std::function<array_1d<double, 3>(const GeometryType&, const GeometryType::CoordinatesArrayType&, const double)> retrieve_normal_area_normal =
     [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) -> array_1d<double, 3> {return Coefficient * rGeometry.Normal(rLocalCoordinates);};
 
     const auto* p_retrieve_normal = ConsiderUnitNormal ? &retrieve_normal_unit_normal : &retrieve_normal_area_normal;
 
-    // TODO: Use TLS in ParallelUtilities
-    #pragma omp parallel for firstprivate(aux_coords)
-    for (int i = 0; i < static_cast<int>(r_entities_array.size()); ++i) {
+    // Declare auxiliar coordinates
```

Comment:
I thought we were over this.

```suggestion
    // Declare auxiliary coordinates
```

Reply:
![](https://i.makeagif.com/media/3-03-2022/q_YbaM.gif)